semantics/psc-trees-unary.ads

```diff
@@ -48,6 +48,8 @@ package PSC.Trees.Unary is
       Operand : Optional_Tree;
    end record;
 
+   type Tree_Ptr is access all Tree'Class;
```

Comment:
This declaration should probably be removed, since it is redundant with the similar declaration on line 84 of psc-trees.ads